tests/unittests/tests-gnrc_mac_internal/Makefile.include

```diff
@@ -0,0 +1,2 @@
+USEMODULE += gnrc_priority_pktqueue
+USEMODULE += gnrc_mac
```

Comment:
@miri64 Added [this](https://github.com/RIOT-OS/RIOT/pull/6074/files#diff-0bece8cbc8c4b720a8b00b3e813897ceR3), not sure whether this is what you want. ;-)